bip-0179.mediawiki

```diff
@@ -0,0 +1,56 @@
+<pre>
+  BIP: 179
+  Title: Name for payment recipient identifiers
+  Author: Emil Engler <me@emilengler.com>
+          MarcoFalke <falke.marco@gmail.com>
+          Luke Dashjr <luke+bip@dashjr.org>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0
```

Comment:
Undefined license CC0 in bip-0179.mediawiki at scripts/buildtable.pl line 154, <$F> line 10.

Reply:
Should be `CC0-1.0`

Reply:
Fixed